### PR TITLE
Und 720

### PR DESCRIPTION
**Pull request by Ashish-srij**
header issues

### CODE REVIEW

File: stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.scss [hunk 1]
```diff
@@ -119,24 +121,34 @@
...
+        &:last-child {
+          justify-content: flex-start;
+          padding-left: 0;
+          padding-right: 6.9375rem;
+        }
+  
+        li {
+          margin: $spacing-12 4.4375rem $spacing-12 0;
+          
+  &:first-child{margin-right: 0;}
```
Comment (shishirphadke) [inline stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.scss:138]: @Ashish-srij Kindly fix alignment and also format it properly.

File: stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/MegaMenu.scss [hunk 2]
```diff
@@ -69,10 +70,13 @@
         line-height: 0;
         list-style-type: none;
         margin-bottom: $spacing-24;
+        text-transform: uppercase;
       }
     }
   }
-
+.mega-title{
```
Comment (shishirphadke) [inline stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/MegaMenu.scss:77]: @Ashish-srij Kindly fix alignment and also format it properly.

File: stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.scss (hunk 1 above)
Comment (Ashish-srij) [reply stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.scss]: @shishirphadke Done please check.

File: stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/MegaMenu.scss (hunk 2 above)
Comment (Ashish-srij) [reply stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/MegaMenu.scss]: @shishirphadke Done Please check.

**Pull request merged**